frontend/src/components/Snackbar.tsx

```diff
@@ -0,0 +1,43 @@
+import clsx from "clsx";
+
+import { ReactNode } from "react";
+import { Button } from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+type SnackbarProps = {
+  children: ReactNode;
+  close: () => void;
+  isVisible: boolean;
+};
+
+const Snackbar = ({ children, close, isVisible = false }: SnackbarProps) => {
+  return (
+    <div
+      aria-hidden={!isVisible}
+      role="status"
+      data-testid="snackbar"
+      className={clsx(
+        "bg-base-darkest display-flex flex-align-start font-sans-2xs position-fixed padding-2 radius-md right-0 text-left text-white top-0 usa-modal-wrapper",
```

Comment:
> Using a line-break looked cleaner than setting a max-width for this message.

Is this where you tried a max width? Why wouldn't that look as nice as a line break? I fear relying on line breaks could be fragile, since it depends on the content for layout. And on very small screens the content could wrap to additional lines in an awkward way. 

Also wondering if we should use a slightly larger offset than `right-0` and `top-0`

Reply:
Can we try these classes: `maxw-full top-0 right-0 tablet:maxw-mobile-lg tablet:top-2 tablet:right-2`? 

This would make it full width on small screens and limited to the top-right corner on larger screens. I think the line break you're using makes it about the width of `mobile-lg` (480px). 

We'll also need to enable the media queries for the top/right classes: 

```
$right-settings: ( responsive: true ), 
$top-settings: ( responsive: true ), 
```

If we ever want a snackbar that's not limited width, we can refactor this or pass it a prop to make it full-width/limited-width.

Reply:
I added those classes. 

> This would make it full width on small screens

Did you mean `width-full`? It isn't full width, just won't be larger than the screen on mobile:

![image](https://github.com/user-attachments/assets/b6e67646-8979-4874-865f-7a424c123c89)

Reply:
Good catch! I was assuming the content would be long enough to force it to full width. You're right, it'll look better on small screens if it's full width regardless of content length.